config/routes.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
+  # resources :confirmations
   resources :sessions
-  resources :users
-  root "users#new"
+  resources :users   
+  resources :password_resets
+  resources :registered_applications
```

Comment:
Consider limiting `resources` routes by use of the `:only` option so there are no excess/unused routes.